frontend/app/src/pages/main/tenant-settings/components/revoke-token-form.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@/components/ui/button';
+import { Spinner } from '@/components/ui/loading.tsx';
+import {
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/ui/dialog';
+import { APIToken } from '@/lib/api';
+
+interface RevokeTokenFormProps {
+  className?: string;
+  onSubmit: (apiToken: APIToken) => void;
+  onCancel: () => void;
+  apiToken: APIToken;
+  isLoading: boolean;
+}
+
+export function RevokeTokenForm({ className, ...props }: RevokeTokenFormProps) {
+  return (
+    <DialogContent className="w-fit max-w-[80%] min-w-[500px]">
+      <DialogHeader>
+        <DialogTitle>Delete invite</DialogTitle>
+      </DialogHeader>
+      <div>
+        <div className="text-sm text-foreground mb-4">
+          Are you sure you want to revoke the API token {props.apiToken.name}?
```

Comment:
Clarify impact on action

```suggestion
          Are you sure you want to revoke the API token {props.apiToken.name}? This action will immediately prevent any services running with this token from dispatching events or executing steps.
```